app/pages/notifications/notification-section.jsx

```diff
@@ -177,9 +177,9 @@ export default class NotificationSection extends Component {
       avatar = <img src={src} className="notification-section__img" alt="Project Avatar" />;
     }
     return (
-      <Link to={this.props.slug ? `/projects/${this.props.slug}` : '/'}>
+      <a href={this.props.slug ? `https://www.zooniverse.org/projects/${this.props.slug}` : 'https://www.zooniverse.org'}>
```

Comment:
I'm not totally sure about this one. Should it be ` <a href={this.props.slug ? `/projects/${this.props.slug}` : 'https://www.zooniverse.org'}>` because the notifications page is PFE and if there is a slug prop then you'd want to stay on PFE?

Reply:
Yes I think this can remain as Link. For FEM projects the [PFE rerouting](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/monorepoUtils.js) will kick in.

Reply:
Yes I think this can remain as Link. For FEM projects the [PFE rerouting](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/monorepoUtils.js) will kick in. But this works, and eventually when all projects migrate we'll need to make a handful of similar changes.